time/src/serde/mod.rs

```diff
@@ -27,24 +27,34 @@ use core::marker::PhantomData;
 #[cfg(feature = "serde-human-readable")]
 use serde::ser::Error as _;
 use serde::{Deserialize, Deserializer, Serialize, Serializer};
-/// Generate a custom serializer and deserializer from the provided string.
+/// Generate a custom serializer and deserializer from a format string or an existing format.
 ///
 /// The syntax accepted by this macro is the same as [`format_description::parse()`], which can
 /// be found in [the book](https://time-rs.github.io/book/api/format-description.html).
 ///
 /// # Usage
 ///
-/// Invoked as `serde::format_description!(mod_name, Date, "<format string>")`. This puts a
-/// module named `mod_name` in the current scope that can be used to format `Date` structs. A
-/// submodule (`mod_name::option`) is also generated for `Option<Date>`. Both modules are only
-/// visible in the current scope.
+/// Invoked as `serde::format_description!(mod_name, Date, FORMAT)` where `FORMAT` is either a
+/// `"<format string>"` or something that implements
+#[cfg_attr(
+    feature = "formatting",
+    doc = "[`Formattable`](crate::formatting::Formattable)"
+)]
+#[cfg_attr(not(feature = "formatting"), doc = "`Formattable`")]
+/// and/or
+#[cfg_attr(feature = "parsing", doc = "[`Parsable`](crate::parsing::Parsable)")]
+#[cfg_attr(not(feature = "parsing"), doc = "`Parsable`")]
```

Comment:
It would be better to only show the traits that need to be implemented, rather than showing a trait that isn't available with the current feature set.